libbeat/common/kubernetes/util.go

```diff
@@ -30,10 +30,25 @@ import (
 	"k8s.io/client-go/tools/clientcmd"
 	clientcmdapi "k8s.io/client-go/tools/clientcmd/api"
 
+	"github.com/pkg/errors"
+
 	"github.com/elastic/beats/v7/libbeat/logp"
 )
 
-const defaultNode = "localhost"
+const namespaceFilePath = "/var/run/secrets/kubernetes.io/serviceaccount/namespace"
+
+type discoveryUtils struct {
+	eDisc       HostDiscovery
```

Comment:
I'd really like a different name for this `eDisc` field. One that clearly states that it's used for host discovery purposes.

Reply:
I second this, as I find very difficult to understand what is supposed to mean. Some documentation may also works.